aziotctl/aziotctl-common/src/status.rs

```diff
@@ -0,0 +1,125 @@
+use std::fmt;
+use std::process::Command;
+
+use crate::ServiceDefinition;
+
+#[allow(clippy::module_name_repetitions)]
+pub fn get_status(processes: &[&ServiceDefinition]) {
+    let results: Vec<ServiceStatus<'_>> = processes
+        .iter()
+        .map(|process| ServiceStatus {
+            service_name: process.service,
+            service_status: read_status(process.service),
+            sockets: process
+                .sockets
+                .iter()
+                .map(|socket| SocketStatus {
+                    socket_name: socket,
+                    socket_status: read_status(socket),
+                })
+                .collect(),
+        })
+        .collect();
+
+    if results.iter().any(|s| !s.ok()) {
+        for result in &results {
+            println!("{}: {}", result.service_name, result.service_status);
+            for socket in &result.sockets {
+                println!("{}: {}", socket.socket_name, socket.socket_status);
+            }
+            println!();
+        }
+
+        for result in &results {
```

Comment:
Is printing the logs here part of Micah's spec?

Reply:
yep